gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/remote-connection.js

```diff
@@ -33,9 +33,11 @@ class RemoteConnection {
   /**
    * @abstract
    * @param {Bytecode} bytecode
+   * @param {String} op Operation to perform, defaults to bytecode.
+   * @param {Object} args The arguments for the operation. Defaults to
```

Comment:
Missing jsdoc :)

"Defaults to an associative array containing values for "aliases" and "gremlin" keys.

Reply:
Yep missed that, sorry :)